src/app/courses/view-courses/view-courses.component.html

```diff
@@ -0,0 +1,25 @@
+<mat-toolbar>
+  <a mat-icon-button routerLink="/courses"><mat-icon>arrow_back</mat-icon></a>
+</mat-toolbar>
+
+<div class="space-container">
+  <mat-toolbar class="primary-color font-size-1">
+    <span class="margin-lr-3"><h3>{{courseDetail.courseTitle}}</h3></span>
+    <span class="toolbar-fill"></span>
+  </mat-toolbar>
+  <div class="view-container" >
+    <div class="course-detail bg-light-grey">
+      <p>Course Leader: {{courseDetail.courseLeader}} </p>
+      <p>Description:</p><textarea readonly>{{courseDetail.description}}</textarea>
+      <p>Subject Level: {{courseDetail.subjectLevel}}</p>
+      <p>Grade Level: {{courseDetail.gradeLevel}}</p>
+      <p>Language of Instruction: {{courseDetail.languageOfInstruction}}</p>
+      <p>Method: {{courseDetail.method}}</p>
+      <p>Frequency/Days: {{courseDetail.day}}</p>
+      <p>Schedule - 
+        Date: {{courseDetail.startDate}} &nbsp;
```

Comment:
@paulbert , @lmmrssa , @dogi we need to have same date format for all over the pages. So, what should be the date format ?
For example , 14th Feb, 2018

Reply:
that is something we can set based on country or date locale. Or we could create a config which will select default date format.
For now keep US standard date format

Reply:
Please use the [DatePipe](https://angular.io/api/common/DatePipe) as it will link with the locale for i18n translation.